src/assets/styles/variables.scss

```
@@ -1,6 +1,3 @@
 $font-stack: Roboto;
-$primary-color: #5151d5;
-
-.color-primary {
-  color: $primary-color !important;
-}
+$--color-primary: #5957d6;
+$--color-white: #fff;
```

```suggestion
$primary--color: #5957d6;
$white--color: #fff;
```
declare css variables like this.